mappings/net/minecraft/world/chunk/WorldChunk.mapping

```diff
@@ -60,11 +60,20 @@ CLASS net/minecraft/class_2818 net/minecraft/world/chunk/WorldChunk
 		ARG 1 levelTypeProvider
 	METHOD method_12208 createBlockEntity (Lnet/minecraft/class_2338;)Lnet/minecraft/class_2586;
 		ARG 1 pos
-	METHOD method_12210 getEntities (Ljava/lang/Class;Lnet/minecraft/class_238;Ljava/util/List;Ljava/util/function/Predicate;)V
+	METHOD method_12210 collectEntitiesByClass (Ljava/lang/Class;Lnet/minecraft/class_238;Ljava/util/List;Ljava/util/function/Predicate;)V
+		COMMENT Collects a list of entities of some runtime type and appends them to the given list. The type of
+		COMMENT the collected entities will be a subclass of the given class.
```

Comment:
"a subclass of" -> "instances of"

Reply:
nope, interfaces don't work here per @kvverti

Reply:
The method does have a type constraint preventing an interface from being used already:
`<T extends Entity> void getEntities(Class<? extends T> entityClass [...]`

Reply:
Ah I missed that! I can probably remove the exception specification too, then.